File: content/examples/other/leetcode/problems/search-in-rotated-sorted-array/example-1/Solution.java
public class Solution {
  static int searchLinear(int[] nums, int target) {
    for (int i = 0; i < nums.length; i += 1) {
      if (nums[i] == target) return i;
    }
    return -1;
  }

  public static void main(String[] args) {
    System.out.println(searchLinear(new int[] {4, 5, 6, 7, 0, 1, 2}, 0));
  }
}

File: content/examples/other/leetcode/problems/search-in-rotated-sorted-array/example-1/solution.js
function searchLinear(nums, target) {
  for (let i = 0; i < nums.length; i += 1) {
    if (nums[i] === target) return i;
  }
  return -1;
}

if (require.main === module) {
  console.log(searchLinear([4, 5, 6, 7, 0, 1, 2], 0));
}

module.exports = { searchLinear };
